src/locales/en/translation.json

```diff
@@ -479,6 +479,15 @@
     "coming": "Coming",
     "soon": "soon"
   },
+  "paratimes": {
+    "active": "Active",
+    "activeNodes": "{{nodes}} active",
+    "blockNumber": "Block number:",
+    "inactive": "Suspended",
```

Comment:
@donouwens this implies that Cipher runtime is suspended, not just that indexer doesn't support it.
![image](https://github.com/oasisprotocol/explorer/assets/3758846/aac6ddbc-2757-4cc6-9748-4d8640fd604e)

do we need more status options, or is this just to demo how suspended ones will look like, and we should change Cipher to "Active" but with no data?

Reply:
@donouwens, we don't want to show new ParaTime Pontus-X here, right?

Reply:
I have changed "Suspended" to "Coming soon" for now. My preference would be to remove status row entirely.

Reply:
I expect in the future we are supposed to get data from API about the *actual* status of runtimes

Reply:
Oh I thought it's rather avail/or not avail in Explorer, but yeah useRuntimeFreshness makes sense too. Will add a ticket to address this in the future.